Source/Initialization/WarpXInitData.cpp

```diff
@@ -431,6 +436,11 @@ WarpX::InitData ()
         if (electrostatic_solver_id == ElectrostaticSolverAlgo::LabFrameElectroMagnetostatic)
             ComputeMagnetostaticField();
 
+        // Set up an invariant that any code besides the field solver
```

Comment:
I still have a bit of trouble understanding this comment.
Was it maybe meant to be like this ("so" or "such" missing)?
```suggestion
        // Set up an invariant so that any code besides the field solver
```

Reply:
It's written as I meant it, but the wording may be too heavily 'academic CS'. The 'invariant' is "any code besides the solver will see the composite values".

I'm happy to reword it as requested if you find that more comprehensible.

Reply:
Thanks Phil. If that's okay with you, yes, I would be in favor of rewording it a little bit.